.github/workflows/deploy.yml

```diff
@@ -0,0 +1,69 @@
+name: Deploy
+
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        type: string
+        required: true
+        description: "Name of the artifact to deploy"
+      docs-output-prefix:
+        type: string
+        required: true
+        description: "Path prefix where the docs will be generated"
+      dry-run:
+        type: boolean
+        required: true
+        description: "Dry run (doesn't actually publish anything)"
+    secrets:
+      NUGET_API_KEY:
+        required: true
+      BOT_GITHUB_TOKEN:
+        required: true
+
+env:
+  NUGET_SERVER_URL: https://www.nuget.org/
+  NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+
+jobs:
+
+  deploy:
+    needs: build
```

Comment:
Stupid mistake, I forgot to remove this. There must be at least one job with no dependency, otherwise the workflow is not valid.